docs/firmware-design.rst

```diff
@@ -2433,6 +2433,29 @@ This Architecture Extension is targeted when ``ARM_ARCH_MAJOR`` == 8 and
    table entries for a given stage of translation for a particular translation
    regime.
 
+ARMv7
+~~~~~
+
+This Architecture Extension is targeted when ``ARM_ARCH_MAJOR`` == 7.
+
+There are several ARMv7 extensions available. Obviously the TrustZone
+extension is mandatory to support the ARM Trusted Firmware bootloader
+and runtime services.
+
+Platform implementing an ARMv7 system can to define from its target
+Cortex-A architecture through ``ARMV7_CORTEX_A<X> = yes``. Defined
+targets are:
+* ``ARMV7_CORTEX_A5=yes`` for a Cortex-A5 target.
+* ``ARMV7_CORTEX_A7=yes`` for a Cortex-A7 target.
+* ``ARMV7_CORTEX_A9=yes`` for a Cortex-A9 target.
+* ``ARMV7_CORTEX_A15=yes`` for a Cortex-A15 target.
```

Comment:
Unless you're going to enumerate all possible CPUs here I suggest you just use one of the above as an example.